packages/test-utils/src/reporter.ts

```diff
@@ -20,6 +20,7 @@ import { log, warn } from "./logging";
 import { getMetadataFilePath } from "./metadata";
 import { pingTestMetrics } from "./metrics";
 import { buildTestId, generateOpaqueId } from "./testId";
+import { Logger, getAuthIds } from "@replayio/observability-node";
```

Comment:
I don't think we can merge+release until we have decided on the sharing strategy.